redfish-core/lib/pcie.hpp

```diff
@@ -482,6 +598,40 @@ inline void requestRoutesSystemPCIeDevice(App& app)
                     return;
                 }
             });
+
+    BMCWEB_ROUTE(app, "/redfish/v1/Systems/system/PCIeDevices/<str>/")
+        .privileges(redfish::privileges::patchPCIeDevice)
+        .methods(boost::beast::http::verb::patch)(
+            [](const crow::Request& req,
+               const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
+               const std::string& device) {
+                std::optional<nlohmann::json> oem;
+                if (!json_util::readJson(req, asyncResp->res, "Oem", oem))
+                {
+                    return;
+                }
+
+                if (oem)
+                {
+                    std::optional<nlohmann::json> ibmOem;
+                    if (!redfish::json_util::readJson(*oem, asyncResp->res,
+                                                      "IBM", ibmOem))
+                    {
+                        return;
+                    }
+
+                    if (ibmOem)
```

Comment:
if (!ibmOem)
        {
            BMCWEB_LOG_ERROR << "Property Missing ";
            messages::propertyMissing(asyncResp->res, "IBM");
            return;
        }

Reply:
Actually readJson handles this pretty well